assets/vue/convos-connection-editor.vue

```diff
@@ -68,6 +71,7 @@ module.exports = {
   data: function() {
     return {
       advanced:         false,
+      on_connect:       "",
```

Comment:
Let's use CamelCase (onConnectCommands) in javascript, unless it maps directly to an API variable, such as "dialog_id".